webpack.config.js

```diff
@@ -99,7 +100,10 @@ module.exports = (env, args = {}) => {
       cache: !isProduction, // Improves linting performance
     }),
 
-    new webpack.DefinePlugin(globalVariables), // Set node variables.
+    new webpack.DefinePlugin({
+      globalVariables, // Set node variables
+      "process.env": JSON.stringify(process.env),
```

Comment:
todo: this creates two problems:

1. You overwrite the `process.env` from `globalVariables.
2. Using the `process.env` variable adds many additional variables, that are not required.

Please extend the `globalVariables` instead, by spreading the imported values:

```js
const envVariables = require('dotenv').config({ path: './.env' }).parsed;

// ... stuff

const globalVariables = {
  'process.env': {
    ...envVariables,
    foo: 'baa',
  }
}
```